backend/tests/v5/unit/models/metadata.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { src } = require('../../helper/path');
+
+const { generateRandomString } = require('../../helper/services');
+
+const Metadata = require(`${src}/models/metadata`);
+const db = require(`${src}/handler/db`);
+const { templates } = require(`${src}/utils/responseCodes`);
+
+const testGetMetadataById = () => {
+	describe('Get metadata by Id', () => {
+		test('should return error if the metadata does not exist', async () => {
+			const teamspace = generateRandomString();
+			const model = generateRandomString();
+			const metadataId = generateRandomString();
+			jest.spyOn(db, 'findOne').mockResolvedValue(undefined);
```

Comment:
use the once functions instead of the intention is for it to only happen once
```suggestion
			jest.spyOn(db, 'findOne').mockResolvedValueOnce(undefined);
```